tests/test_transforms/test_feature_selection/test_gale_shapley_transform.py

```diff
@@ -654,8 +654,8 @@ def test_right_number_features_with_integer_division(ts_with_exog_galeshapley):
     top_k = len(ts_with_exog_galeshapley.segments)
     transform = GaleShapleyFeatureSelectionTransform(relevance_table=StatisticsRelevanceTable(), top_k=top_k)
 
-    transform.fit(ts_with_exog_galeshapley.to_pandas())
-    df = transform.transform(ts_with_exog_galeshapley.to_pandas())
+    transform.fit(ts_with_exog_galeshapley)
+    ts = transform.transform(ts_with_exog_galeshapley)
```

Comment:
How did this test pass before?

Reply:
It is the artefact of rebase, is you see the PR with etna_v2 it fails there